src/PsrContainerExtension.php

```diff
@@ -38,7 +41,11 @@ public function load(ContainerBuilder $container, array $config)
     {
         $container->setParameter('roave.behat.psr.container.included.file', $config['container']);
 
-        $loader = new YamlFileLoader($container, new FileLocator(__DIR__));
-        $loader->load('services.yml');
+        $definition = new Definition(ContainerInterface::class, ["%roave.behat.psr.container.included.file%"]);
```

Comment:
ID is unique, `class` should correspond to the actual type, but with factory services like this one it's not actually checked (in services that directly instantiate a class it's the class to instantiate).

The only thing this affects is static analysis/autocomplete so it should be set to whatever you want those tools to infer about`$this->get($serviceId)`'s return type.